privacyidea/api/validate.py

```diff
@@ -416,10 +416,11 @@ def check():
                     # additional attributes
                     for k, v in ui.items():
                         result["attributes"][k] = v
-
+    serials = ",".join([challenge_info["serial"] for challenge_info in details["multi_challenge"]]) \
+        if 'multi_challenge' in details else details.get('serial')
     g.audit_object.log({"info": log_used_user(user, details.get("message")),
                         "success": success,
-                        "serial": serial or details.get("serial"),
+                        "serial": serials,
```

Comment:
We should also add a testcase for this and update the frontend code to create the links

Reply:
The frontend code actually handles the serials correctly. Links to the various tokens are generated and work.
![serials2](https://user-images.githubusercontent.com/67051749/188634341-673deee7-c483-4dd3-86df-68a13d7211f9.png)